### PR TITLE
Rename add command

### LINKED ISSUE #105: Rename AddCommand to CreateMemberCommand

### PR DESCRIPTION
**Pull request by dylkaw**
Closes #105.

Made the following changes:
- Deleted `AddCommand` and `AddCommandParser` classes
- Created `CreateMemberCommand` and `CreateMemberCommandParser`
- Deleted `AddCommandIntegrationTest`, `AddCommandTest` and `AddCommandParserTest` test files
- Created `CreateMemberCommandIntegrationTest`, `CreateMemberCommandTest` and `CreateMemberCommandParserTest`
- Renamed all instances of `AddCommand` and `AddCommandParser` to `CreateMemberCommand` and `CreateMemberCommandParser` respectively

### CODE REVIEW

**Comment by wr1159** [comment]
LGTM overall, I saw you change one comment to use CCACommander, should we have another PR to rename all AB instances to CCACommander?

Also one small thing, should we rename the header comment here?

File: src/main/java/seedu/address/model/Model.java [hunk 1]
```diff
@@ -68,7 +68,7 @@ public interface Model {
      * Adds the given member.
```
Comment (wr1159) [inline src/main/java/seedu/address/model/Model.java]:
```suggestion
     * Creates the given member.
```
should we change to this?

File: src/test/java/seedu/address/logic/LogicManagerTest.java [hunk 2]
```diff
@@ -165,11 +165,11 @@ public void saveAddressBook(ReadOnlyAddressBook addressBook, Path filePath)
         logic = new LogicManager(model, storage);
 
         // Triggers the saveAddressBook method by executing an add command
```
Comment (wr1159) [inline src/test/java/seedu/address/logic/LogicManagerTest.java]:
```suggestion
        // Triggers the saveAddressBook method by executing a create command
```

**Review by wr1159 — COMMENTED** [review]
LGTM overall, I saw you change one comment to use CCACommander, should we have another PR to rename all AB instances to CCACommander?

File: src/main/java/seedu/address/model/Model.java (hunk 1 above)
Comment (dylkaw) [reply src/main/java/seedu/address/model/Model.java]: Yes, thanks for spotting it!

**Comment by dylkaw** [comment]
> LGTM overall, I saw you change one comment to use CCACommander, should we have another PR to rename all AB instances to CCACommander?

Sounds good, I will open an issue.

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java [hunk 3]
```diff
@@ -38,8 +38,9 @@ public class AddressBookParserTest {
     @Test
     public void parseCommand_add() throws Exception {
```
Comment (SelwynAng) [inline src/test/java/seedu/address/logic/parser/AddressBookParserTest.java]: Should this method name be changed to `parseCommand_create()`?

**Review by SelwynAng — COMMENTED** [review]
LGTM overall! Just one small nit.

File: src/test/java/seedu/address/logic/parser/AddressBookParserTest.java (hunk 3 above)
Comment (dylkaw) [reply src/test/java/seedu/address/logic/parser/AddressBookParserTest.java]: Yes it should be! Thanks for spotting it!

**Pull request merged**